frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -0,0 +1,492 @@
+                .join(", ")
+            : null}
+        </InfoValue>
+      </InfoItem>
+      <InfoItem>
+        <RiGlobalLine />
+        <InfoLabel>Languages:</InfoLabel>
+        <InfoValue>
+          {run.languages?.length
+            ? run.languages
+                .map((language) => ISO6391.getName(language.substring(0, 2)))
+                .join(", ")
+            : null}
+        </InfoValue>
+      </InfoItem>
+    </InfoItems>
+  )
+}
+
+const getCtaPrefix = (type: ResourceTypeEnum) => {
+  if (type === ResourceTypeEnum.Podcast) {
+    return "Listen to"
+  } else {
+    return "Take"
+  }
+}
+
+const LearningResourceExpanded: React.FC<LearningResourceExpandedProps> = ({
+  resource,
+  imgConfig,
+}) => {
+  const [selectedRun, setSelectedRun] = useState(resource?.runs?.[0])
+
+  const multipleRuns = resource?.runs && resource.runs.length > 1
+
+  useEffect(() => {
+    if (resource) {
+      setSelectedRun(resource!.runs![0])
+    }
+  }, [resource])
+
+  const onDateChange = (event: SelectChangeEvent) => {
+    const run = resource?.runs?.find(
+      (run) => run.id === Number(event.target.value),
+    )
+    if (run) setSelectedRun(run)
+  }
+
+  const DateSection = ({ hide }: { hide?: boolean }) => {
+    if (hide) {
+      return null
+    }
+    if (!resource) {
+      return <Skeleton height={40} style={{ marginTop: 0, width: "60%" }} />
+    }
+
+    if (
+      [ResourceTypeEnum.Course, ResourceTypeEnum.Program].includes(
+        resource.resource_type as "course" | "program",
+      ) &&
+      multipleRuns
+    ) {
+      return (
+        <Date>
+          <DateLabel>Start Date:</DateLabel>
+          <SelectField
+            label={null}
+            value={selectedRun?.id as unknown as string}
+            onChange={onDateChange}
+          >
+            {resource.runs!.map((run) => (
+              <MenuItem key={run.id} value={run.id}>
+                {formatDate(run.start_date!, "MMMM DD, YYYY")}
+              </MenuItem>
+            ))}
+          </SelectField>
+        </Date>
+      )
+    }
+
+    const isOcw =
+      resource.resource_type === ResourceTypeEnum.Course &&
+      resource.platform?.code === PlatformEnum.Ocw
+
+    const nextStart = resource.next_start_date || selectedRun?.start_date
+
+    if (!isOcw && !nextStart && !(selectedRun?.semester && selectedRun?.year)) {
+      return <NoDateSpacer />
+    }
+
+    return (
+      <DateSingle>
+        <DateLabel>{isOcw ? "As taught in:" : "Start Date:"}</DateLabel>
+        {isOcw
+          ? `${selectedRun?.semester} ${selectedRun?.year}`
+          : formatDate(nextStart!, "MMMM DD, YYYY")}
+      </DateSingle>
+    )
+  }
+
+  const isVideo = resource && resource.resource_type === ResourceTypeEnum.Video
+
+  return (
+    <Container padTop={isVideo}>
+      <DateSection hide={isVideo} />
+      <ImageSection resource={resource} config={imgConfig} />
+      <CallToActionSection resource={resource} />
```

Comment:
We may need to get the URL from the runs... Both the resource itself and runs (for courses + programs) have a url. 

However, on my local data, run URLs are either null or always the same as the resource URL. Will check RC/prod after I get off this bus.

cc @mbertrand